base_layer/core/src/mining/miner.rs

```diff
@@ -203,6 +207,10 @@ impl<B: BlockchainBackend> Miner<B> {
 
     /// function, temp use genesis block as template
     pub async fn get_block(&mut self, block: NewBlockTemplate) -> Result<Block, MinerError> {
+        trace!(
+            target: LOG_TARGET,
+            "Asking node to complete newly mined block from provided template"
```

Comment:
```suggestion
            "Asking node to complete block from provided template"
```
The block is not yet mined at this stage. 

Block mine processes is as follows: 

1. Basenode created template block
2. Miner adds Coinbase UTXO
3. Basenode calculates MMR roots to finish block creation
4. Miner mines block

Reply:
Added these comments to the docstring

Reply:
Also updated the log message